functorch/csrc/BatchRulesViews.cpp

```diff
@@ -272,6 +272,17 @@ std::tuple<Tensor, optional<int64_t>> select_batching_rule(const Tensor& self, o
   return std::make_tuple(result, 0);
 }
 
+std::tuple<Tensor, optional<int64_t>> _reshape_alias_batch_rule(const Tensor& self, optional<int64_t> bdim, const IntArrayRef shape, const IntArrayRef strides) {
+  (void) strides;
+  if (!bdim) {
+    return std::make_tuple(self.reshape(shape), nullopt);
+  }
```

Comment:
The way we do dispatches means that we don't actually need this case: if none of the inputs have bdims, then the batching rule doesn't get called. If you want to assert this you could add a TORCH_INTERNAL_ASSERT(bdim.has_value()).